.travis.yml

```diff
@@ -90,5 +90,9 @@ script:
     if [[ "$NPM_TESTS" == "1" ]]; then
       npm test
     else
+      if [[ "$TRAVIS_PHP_VERSION" != "5.2" ]]; then
```

Comment:
Pro-tip: only run PHPCS once per build preferably on a high PHP version.

You can use a `PHPCS=1` environment variable to achieve this easily, similar to what's done for `NPM_TESTS`.

All sniffs are unit tested against all supported PHP versions, so running PHPCS against every PHP version is redundant.